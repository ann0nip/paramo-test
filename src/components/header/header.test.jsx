import { cleanup, render, screen } from '@testing-library/react';
import Header from './header.component';

afterEach(cleanup);
describe('Header Component', () => {
    it('should take a snapshot', () => {
        const { asFragment } = render(<Header />);

        expect(asFragment()).toMatchSnapshot();
    });

    it('should contain the title POKEDEX', () => {
        render(<Header />);

        screen.getByText('POKEDEX');
    });
});
