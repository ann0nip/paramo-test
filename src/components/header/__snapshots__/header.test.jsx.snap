// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header Component should take a snapshot 1`] = `
<DocumentFragment>
  <header
    class="MuiPaper-root MuiPaper-elevation MuiPaper-elevation4 MuiAppBar-root MuiAppBar-colorPrimary MuiAppBar-positionStatic css-hip9hq-MuiPaper-root-MuiAppBar-root"
  >
    <div
      class="MuiContainer-root MuiContainer-maxWidthXl css-19r6kue-MuiContainer-root"
    >
      <div
        class="MuiToolbar-root MuiToolbar-gutters MuiToolbar-regular css-15dd1oo-MuiToolbar-root"
      >
        <a
          class="MuiTypography-root MuiTypography-h4 MuiTypography-gutterBottom MuiTypography-noWrap css-a7ux5u-MuiTypography-root"
          href="/"
        >
          POKEDEX
        </a>
      </div>
    </div>
  </header>
</DocumentFragment>
`;
